modules/ml/src/main/java/org/apache/ignite/ml/environment/DefaultLearningEnvironmentBuilder.java

```diff
@@ -54,8 +60,11 @@
         loggingFactory = constant(NoOpLogger.factory());
         seed = constant(new Random().nextLong());
         rngSupplier = p -> new Random();
+        dataTtl = DEFAULT_TTL;
```

Comment:
When I read "dataTtl = INFINITY_TTL" I can understand what you mean. In case of "DEFAULT_TTL"-name I can't